features/0434-outofband/README.md

```diff
@@ -0,0 +1,465 @@
+
+#### Error Message Example
+
+```jsonc
+{
+  "@type"            : "https://didcomm.org/oob-invitation/%VER/problem_report",
+  "@id"              : "5678876542345",
+  "~thread"          : { "pthid": "<@id of the OutofBand message>" },
+  "description"      : {
+                          "en": "The request has expired.",
+                          "code": "expired-request"
+                       },
+  "impact"           : "thread"
+}
+```
+
+An equivalent "expired-invitation" problem-code is also required.
+
+See the [problem-report](https://github.com/hyperledger/aries-rfcs/tree/master/features/0035-report-problem) protocol for details on the items in the example.
+
+### Flow Overview
+
+In an out of band message the _sender_ gives information to the _receiver_ about the kind of DIDComm protocol response messages it can handle and how to deliver the response. The receiver uses that information to determine what DIDComm protocol/message to use in responding to the sender, and (from the service item) how to deliver the response to the sender.
+
+The handling of the response is specified by the protocol used.
+
+> To Do: Make sure that the following remains in the DID Exchange/Connections RFCs
+>
+> Any Public DID that expresses support for DIDComm by defining a  [`service`](https://w3c.github.io/did-core/#service-endpoints) that follows the [DIDComm conventions](../0067-didcomm-diddoc-conventions/README.md#service-conventions) serves as an implicit invitation. If an _invitee_ wishes to connect to any Public DID, they need not wait for an out of band invitation message. Rather, they can designate their own label and initiate the appropriate protocol (e.g. [0160-Connections](https://github.com/hyperledger/aries-rfcs/blob/9b0aaa39df7e8bd434126c4b33c097aae78d65bf/features/0160-connection-protocol/README.md#0-invitation-to-connect) or [0023-DID-Exchange](https://github.com/hyperledger/aries-rfcs/blob/9b0aaa39df7e8bd434126c4b33c097aae78d65bf/features/0023-did-exchange/README.md#0-invitation-to-exchange)) for establishing a connection.
+
+### Standard Out of Band Message Encoding
+
+Using a standard out of band message encoding allows for easier interoperability between multiple projects and software platforms. Using a URL for that standard encoding provides a built in fallback flow for users who are unable to automatically process the message. Those new users will load the URL in a browser as a default behavior, and may be presented with instructions on how to install software capable of processing the message. Already onboarded users will be able to process the message without loading in a browser via mobile app URL capture, or via capability detection after being loaded in a browser.
+
+The standard out of band message format is a URL with a Base64URLEncoded json object as a query parameter.
+
+The URL format is as follows, with some elements described below:
+
+```text
+https://<domain>/<path>?c_i=<outofbandMessage>
```

Comment:
`c_i` was short for connection invitation. Given the changed nature and naming of this protocol, I think it should be updated to something more appropriate. Perhaps `oob`?

Reply:
Fixed.